src-tauri/tauri.conf.json

```
@@ -32,6 +32,21 @@
                         "cmd": "amap",
                         "args": true
                     },
+                    {
+                        "name": "amass",
+                        "cmd": "amass",
+                        "args": true
+                    },
+                    {
+                        "name": "snmp-check",
+                        "cmd": "snmp-check",
+                        "args": true
+                    },
+                    {
+                        "name": "python3",
+                        "cmd": "python3",
+                        "args": true, 
```

You got an error here.
```suggestion
                        "args": true
```